### PR TITLE
Modified Open Issues badge

### PR DESCRIPTION
**Pull request by lalitmee**
I think, now it should work.

### CODE REVIEW

**Comment by lalitmee** [comment]
Sorry for conflict. I didn't pull after that shifting. I think it is easy to resolve.

**Comment by gauthamzz** [comment]
It's okay resolve conflicts.

**Comment by lalitmee** [comment]
I am not understanding how to resolve this. This is showing something like this. Can you help me @gauthamzz 
screenshot from 2017-12-29 16-57-13

**Comment by gauthamzz** [comment]
Remove <<<< ====and >>>>> part 
Also remove the section you don't want.

File: docs/README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 
 # ANNA
-
-[![GitHub issues](https://img.shields.io/github/issues/bevacqua/awesome-badges.svg)](https://github.com/Anna-Assistant/Anna/issues)
+=======
```
Comment (gauthamzz) [inline docs/README.md]: This ==== part is not needed

**Review by gauthamzz — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (lalitmee) [reply docs/README.md]: I think now it is perfect.

**Comment by lalitmee** [comment]
@gauthamzz Please review the current status of Readme file.

**Comment by lalitmee** [comment]
@gauthamzz should I create another pull request after closing this one?

**Comment by gauthamzz** [comment]
This one is fine

**Comment by lalitmee** [comment]
@gauthamzz Now **open issues** badge is working. 
Is **download from chrom-webstore** badge is fine ?

**Comment by gauthamzz** [comment]
Yeah . I checked them they work great . Thanks

**Comment by lalitmee** [comment]
No need to say thanx. I was interested in contributing, so I did. Welcome.

**Pull request merged**